iceberg/pom.xml

```diff
@@ -31,7 +31,7 @@
         <path.to.iceberg.root>.</path.to.iceberg.root>
         <iceberg-api.version>0.11.0</iceberg-api.version>
         <kryo-shaded.version>4.0.2</kryo-shaded.version>
-        <iceberg.mockito-core.version>1.10.19</iceberg.mockito-core.version>
+        <iceberg.mockito-core.version>3.4.4</iceberg.mockito-core.version>
```

Comment:
How big is this change? Would it worth to separate to a different jira?

Reply:
To bump the version in other places as well I would need to touch 7-8 modules. It's not a trivial change.

Reply:
Do we want to create a different PR for it?
Would it worth the effort?